### PR TITLE
replace link for Date.parse

### PR DESCRIPTION
**Pull request by codehag**
date.parse repo seems to have disappeared

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,8 +50,7 @@ This list contains only stage 1 proposals and higher that have not yet been with
 ### Stage 1
 
 | :rocket: | Proposal                                                                                     | Author                                               | Champion                                             |
-| -------- | -------------------------------------------------------------------------------------------- | ---------------------------------------------------- | ---------------------------------------------------- |
-|          | [`Date.parse` fallback semantics][date-parse]                                                | Morgan Phillips                                      | Morgan Phillips                                      |
+| -------- | -------------------------------------------------------------------------------------------- | ---------------------------------------------------- | ---------------------------------------------------- |                                    |
```
Comment (tschneidereit) [inline README.md]: This line shouldn't be changed. I can't see exactly what the change was, but it broke the table formatting, at least in the `View` preview in the PR.

**Comment by codehag** [comment]
cool, thanks for finding that @ljharb -- i was searching for it as well and wasn't able to find it

**Comment by codehag** [comment]
link has been replaced

**Comment by littledan** [comment]
We've been talking in the Date calls about next steps for this work. I don't think we should consider it dropped. cc @apaprocki

**Comment by littledan** [comment]
Rather than pointing here, probably it makes sense to point to @gibson042's active proposal only, which takes over this line of work.

**Pull request merged**